tests/data/test-data.sql

```diff
@@ -15,10 +15,10 @@ insert into leaderboard (id, technical_name) values
   (1, "global"),
   (2, "ladder_1v1");
 
-INSERT INTO league (id, technical_name, name_key, description_key) VALUES
-  (1, "test_league", "L1", "description_key"),
-  (2, "second_test_league", "L2", "description_key"),
-  (3, "league_without_seasons", "L3", "description_key");
+INSERT INTO league (id, technical_name, image_key, medium_image_key, small_image_key, name_key, description_key) VALUES
+  (1, "test_league", "https://faf.com/", "https://faf.com/medium/", "https://faf.com/small/", "L1", "description_key"),
+  (2, "second_test_league", "https://faf.com/", "https://faf.com/medium/", "https://faf.com/small/", "L2", "description_key"),
```

Comment:
Aren’t they just always the same for every league?

Reply:
That's right. Should I create a completely new table to reduce redundancy? But that also seems a bit inconvenient.

Reply:
In theory this would allow us to have different images for e.g. the Grandmaster 1v1 division in comparison to the Grandmaster 4v4 division. This isn't really planned at the moment, but I could see some benefit in having that flexibility.

Reply:
Hm, I guess that's ok since this will be a pretty small table